WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -267,26 +267,28 @@
 
       <div class="col-sm-offset-2 col-sm-9">
         <div class="row datetimerange">
-          <%= f.input :registration_open, as: :datetime_picker, wrapper: :ranged_datetime, hint: "Note: Registration open and close are in UTC time" %>
+          <%= f.input :registration_open, as: :datetime_picker, wrapper: :ranged_datetime %>
           <%= f.input :registration_close, as: :datetime_picker, wrapper: :ranged_datetime %>
         </div>
       </div>
     </div>
 
     <hr>
-    <%= f.input :remarks, disabled: is_actually_confirmed, hint: "Some additional information you want to pass on to the board. For example, reasons why this competition was requested less than one month away, if applicable." %>
+    <%= f.input :remarks, disabled: is_actually_confirmed %>
 
     <div class="form-group">
       <div class="col-sm-offset-2 col-sm-10">
-        <%= f.button :submit, class: "btn-primary" %>
+        <%= f.button :submit, t('.submit_update_value'), class: "btn-primary" %>
 
         <% if !@competition_admin_view && !is_actually_confirmed && current_user.can_confirm_competition?(@competition) %>
-           <%= f.button :submit, "Confirm", class: "btn-warning", data: { confirm: "Are you sure you're ready to confirm? After confirming, you won't be able to modify any information. Check your email after this to verify that the Board was notified." } %>
+          <%= button_tag(type: 'submit', name: "commit", value: "Confirm", class: "btn btn-warning", data: { confirm: t('.submit_confirm') }) do %>
+            <%= t '.submit_confirm_value' %>
```

Comment:
As we discussed when pair programming a while back, one possible cleanup here would be to change this button to actually send the name: "isConfirmed" and value: "true".

(If we want to do that, it might be worth making that change in a separate PR so it can get merged up asap.)